app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
-<resources>
-    <string name="app_name">AfricanID</string>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+    <string name="app_name">South African ID</string>
+    <string name="tittle">INSERT YOUR DATA</string>
+    <string name="btn_submit">LOAD</string>
+    <string name="txtBx">number</string>
+    <string name="birthDayLabel">Birthday</string>
```

Comment:
Above this file you are using the name of the string variables with this pattern for example btn_submit ( **name of the element first and then "_" the objective of the element nameOfElement_objectiveOfTheElement**) and here you are changing the order of the name  to **objectiveNameOfTheElement,**, could please standardize the names?. Those changes are just for keep the readability of the code.